### PR TITLE
Update navigation with new branding

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

Update navigation with new branding

## QA

Make sure to QA the navigation on small/medium screens as well

## Issue / Card

Fix https://github.com/canonical-web-and-design/web-squad/issues/5643

### CODE REVIEW

**Review by anthonydillon — DISMISSED** [review]
Small content in the code

File: templates/header.html [hunk 1]
```diff
@@ -1,25 +1,27 @@
-<header id="navigation" class="p-navigation">
+<header id="navigation is-dark" class="p-navigation">
   <div class="p-navigation__row">
     <div class="p-navigation__banner">
-      <div class="p-navigation__logo u-vertically-center">
-        <a class="p-navigation__item" href="/" style="color:white;">
-          Netplan
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
```
[the remark this answers is not in the document]
Comment (minkyngkm) [reply templates/header.html]: Thanks! Added 👍

File: templates/header.html [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
       </div>
       <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
       <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
     </div>
-    <nav class="p-navigation__nav">
-      <span class="u-off-screen">
-        <a href="#main-content">Jump to main content</a>
-      </span>
+
+    <nav class="p-navigation__nav" aria-label="navigation">
```
[the remark this answers is not in the document]
Comment (minkyngkm) [reply templates/header.html]: Ah, thanks! Removed.👍

Should we update the examples on https://vanillaframework.io/docs/patterns/navigation as it includes `aria-label` in side `nav` tag 
image

**Comment by minkyngkm** [comment]
Looking good @sowasred2012! One thing I found is when screen is smaller than 620px, the overlay disappears even though the menu is still open. Could it be fixed? :)
ezgif com-gif-maker (10)

**Comment by sowasred2012** [comment]
@minkyngkm good spot - fixed in the global nav repo, and just updated this project with the new version

**Review by steverydz — APPROVED** [review]
LGTM 👍

**Pull request merged**